packages/utils/test/__snapshots__/hashFromSchema.test.ts.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`hashForSchema returns a hash for a more complex schema 1`] = `"-6bbb062a"`;
```

Comment:
I am surprised these are still signed. ideally they would be unsigned but it's not a big deal